R/compareScenarios.R

```diff
@@ -738,11 +738,16 @@ compareScenarios <- function(mif, hist,
              "SE|Electricity|Biomass|w/o CCS (EJ/yr)",
              "SE|Electricity|Solar|CSP (EJ/yr)",
              "SE|Electricity|Solar|PV (EJ/yr)",
-             "SE|Electricity|Wind|Onshore (EJ/yr)",
-             "SE|Electricity|Wind|Offshore (EJ/yr)",
              "SE|Electricity|Hydrogen (EJ/yr)",
              "SE|Electricity|Net Imports (EJ/yr)")
 
+  if ("SE|Electricity|Wind|Offshore (EJ/yr)" %in% magclass::getNames(data, dim = 3)) {
+    items <- append(items, c( "SE|Electricity|Wind|Onshore (EJ/yr)",
```

Comment:
The name "items" is quite general, is there a better name to describe whats actually in this variable?